src/main/cpp/modules/lfo.cpp

```diff
@@ -93,9 +120,26 @@ void LFO::generate()
     else {
         _table->cloneTable( table );
     }
+
+    // ensure tables are unipolar for easy lookup
+
+    if ( WaveUtil::isBipolar( _table->getBuffer(), _table->tableLength )) {
+        WaveUtil::toUnipolar( _table->getBuffer(), _table->tableLength );
+    }
```

Comment:
By making the waveforms unipolar upon pool retrieval/generation, we have less overhead when performing the actual oscillator sweep.